src/api_authentication_app_skeleton/src/actions/mixins/auth/action_helpers.cr

```diff
@@ -0,0 +1,30 @@
+require "jwt"
+
+module Auth::ActionHelpers
+  def sign_in(user : Authentic::PasswordAuthenticatable)
+    context.response.headers.add "Authorization", generate_token(user)
+  end
+
+  def token
+    context.request.headers["Authorization"]
+  end
+
+  def current_user?
+    user_from_token(token.not_nil!)
+  rescue KeyError # Missing hash key: "Authorization" (KeyError)
+    nil
+  rescue JWT::ExpiredSignatureError
+    nil
+  end
+
+  def generate_token(user : Authentic::PasswordAuthenticatable)
+    payload = {"sub" => user.id, "exp" => 14.days.from_now.epoch}
+    JWT.encode(payload, Lucky::Server.settings.secret_key_base, "HS256")
+  end
```

Comment:
What do you think about moving this to the `User` model so the module doesn't need to be included in so many places? It seems this is something that is useful outside HTTP actions o I think the model or a separate `TokenGenerator` object in `app/models` would work. Since it's so simple I think a method in User would be best though